support-frontend/assets/pages/supporter-plus-landing/components/supportOnce.tsx

```diff
@@ -0,0 +1,81 @@
+import { css } from '@emotion/react';
+import {
+	between,
+	from,
+	headline,
+	palette,
+	space,
+	textSans,
+} from '@guardian/source-foundations';
+import { Button } from '@guardian/source-react-components';
+
+const container = css`
+	text-align: center;
+`;
+
+const heading = css`
+	color: ${palette.neutral[7]};
+	margin-top: ${space[4]}px;
+	${headline.xsmall({
+		fontWeight: 'bold',
+	})}
+	${between.tablet.and.desktop} {
+		margin: ${space[4]}px auto 0;
```

Comment:
Think the Figma design does not have 12px here. Perhaps could be -> `		margin: 0 auto;`